modules/docs/mdx/7.0-MIGRATION-GUIDE.mdx

```diff
@@ -538,6 +539,242 @@ main `react` package.
 - `Flex` was promoted from `@workday/canvas-kit-labs/layout` to `@workday/canvas-kit-react/layout`
 - `Stack` was promoted from `@workday/canvas-kit-labs/layout` to `@workday/canvas-kit-react/layout`
 
+## Model Changes
```

Comment:
If you're reviewing this, I recommend viewing the Markdown file here: https://github.com/NicholasBoll/canvas-kit/blob/feat/add-grid-support-temp/modules/docs/mdx/7.0-MIGRATION-GUIDE.mdx#model-changes